app/uk/gov/hmrc/soletraderidentificationfrontend/services/AuditService.scala

```diff
@@ -147,52 +147,52 @@ class AuditService @Inject()(appConfig: AppConfig,
       optFullName <- soleTraderIdentificationService.retrieveFullName(journeyId)
       optDateOfBirth <- soleTraderIdentificationService.retrieveDateOfBirth(journeyId)
       optNino <- soleTraderIdentificationService.retrieveNino(journeyId)
-      optIdentifiersMatch <- soleTraderIdentificationService.retrieveIdentifiersMatch(journeyId)
+      optIsMatch <- soleTraderIdentificationService.retrieveIsMatch(journeyId)
       optAuthenticatorResponse <-
-        optIdentifiersMatch match {
-          case Some(true) =>
+        optIsMatch match {
+          case Some("true") =>
```

Comment:
I'm not certain about this. I suspect the first option will be used even if optNino is empty. Is that what is wanted? I've discussed this with Milly and we are going to create another ticket to investigate this.